packages/aomp-deb/aomp-deb.pacscript

```diff
@@ -1,15 +1,10 @@
 name="aomp-deb"
 gives="aomp"
-version="14.0-2"
+version="14.0-3"
+repology="aomp"
```

Comment:
As stated [here](https://github.com/pacstall/pacstall-programs/discussions/845#discussioncomment-2372145) you can't use `pacup` with `aomp` at the moment.

So I guess remove the variable until in the future an equivalent repology package is available